qtaf-xray-plugin/src/main/java/de/qytera/qtaf/xray/repository/xray/XrayTestRepositoryCloud.java

```diff
@@ -0,0 +1,137 @@
+package de.qytera.qtaf.xray.repository.xray;
+
+import de.qytera.qtaf.core.QtafFactory;
+import de.qytera.qtaf.core.config.exception.MissingConfigurationValueException;
+import de.qytera.qtaf.core.gson.GsonFactory;
+import de.qytera.qtaf.http.RequestBuilder;
+import de.qytera.qtaf.http.WebService;
+import de.qytera.qtaf.xray.dto.request.graphql.GraphQLRequestDto;
+import de.qytera.qtaf.xray.dto.response.steps.XrayCloudTestStepResponseDto;
+import de.qytera.qtaf.xray.dto.response.steps.XrayTestStepResponseDto;
+import de.qytera.qtaf.xray.repository.jira.JiraIssueRepository;
+import jakarta.ws.rs.core.HttpHeaders;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@NoArgsConstructor(access = AccessLevel.PACKAGE)
```

Comment:
Please add DocString

Reply:
Fixed in [7782f10](https://github.com/Qytera-Gmbh/QTAF/pull/89/commits/7782f10a956c9199bb863187a388462b6c0a5224).